accounts/middleware.py

```diff
@@ -0,0 +1,21 @@
+from .models import  MyUser as User
+from django.contrib.auth import authenticate
+from django.utils import timezone
+
+class SetLastVisitMiddleware(object):
+    def __init__(self, get_response):
```

Comment:
Inside of calling `__init__` and `__call__` methods, Can we set the `last_visit` datetime inside of `process_request` method?